pages/get-involved.tsx

```diff
@@ -3,20 +3,96 @@ import {
   Button,
   Container,
   Grid,
+  List,
   Typography,
 } from '@mui/material';
 import React from 'react';
 
 import IndividualPageHead from '../components/helper/IndividualPageHead.tsx';
 import ShareButtons from '../components/helper/ShareButtons.tsx';
 import AccordionBuilder from '../components/layout/AccordionBuilder.tsx';
+import BenefitListItem from '../components/layout/BenefitListItem.tsx';
 import FactCard from '../components/layout/FactCard.tsx';
 import HeroBanner from '../components/layout/HeroBanner.tsx';
 import ImageContainer from '../components/layout/ImageContainer.tsx';
 import PaperSection from '../components/layout/PaperSection.tsx';
 import SectionContainer from '../components/layout/SectionContainer.tsx';
 import content from '../content/get-involved.ts';
 
+function BenefitsOfJoiningUs() {
+  return (
+    <SectionContainer
+      id="benefits-of-joining-us"
+      maxWidth="md"
+    >
+      <Typography variant="h2">
+        {content.benefits.header}
+      </Typography>
+      <PaperSection sx={{
+        bgcolor: 'primary.dark', color: 'primary.contrastText',
+      }}
+      >
+        <Box sx={{
+          pl: 3, py: 4,
+        }}
+        >
+          {content.benefits.body.map((text) => (
+            <Typography
+              key={`benefitsText-${text}`}
+              variant="body1"
+              sx={{ mb: 4, mr: 2 }}
+            >
+              {text}
+            </Typography>
+          ))}
+          <Grid container alignItems="flex-start">
+            <Grid item xs={5}>
+              <Typography variant="body1" paragraph>
+                {content.benefits.listHeader}
+              </Typography>
+            </Grid>
+            <Grid item xs={7} sx={{ display: { xs: 'inherit', sm: 'none' } }}>
+              <ImageContainer
+                src={content.benefits.mobileImgsrc}
+                width={95}
+                height={95}
+                alt=""
+                useImageDimensions
```

Comment:
```suggestion
```
This Image in small devices is breaking.
I believe if we just remove this `useImageDimensions` prop we will be ok.

![image](https://github.com/clearviction-devs/clearviction-wa/assets/80538553/009b1e4b-43d9-41a6-acc8-19d10eb70374)